community-operators/ibm-block-csi-operator-community/1.6.0/bundle-1.6.0.Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM scratch
+
+LABEL operators.operatorframework.io.bundle.channel.default.v1=stable
+LABEL operators.operatorframework.io.bundle.channels.v1=stable
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=ibm-block-csi-operator-community
+
+COPY manifests /manifests/
+COPY metadata /metadata/
+LABEL com.redhat.openshift.versions="v4.7,v4.8"
```

Comment:
Are you claiming here support from v4.7 right? Why add `v4.7,v4.8` and not only v4.7?